pkg/brokerapi/schemas.go

```diff
@@ -39,5 +43,5 @@ type ServiceBindingSchema struct {
 // InputParameters represents a schema for input parameters for creation or
 // update of an API resource.
 type InputParameters struct {
-	Parameters interface{} `json:"parameters,omitempty"`
+	Parameters *runtime.RawExtension `json:"parameters,omitempty"`
```

Comment:
With `interface{}` the actual type of the `Parameters` object is going to be `map[string]interface{}`, and hash map does not guarantee the preserved order by definition.

@luksa if you take a look at the [runtime.RawExtension](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/runtime/types.go#L92) type source code, there is no magic, it's just a wrapper around a raw byte array (see the issue https://github.com/kubernetes-incubator/service-catalog/issues/911 for the justification of its usage in Kubernetes types).

So what you can do instead is to replace `interface{}` with `[]byte`:
```go
type InputParameters struct {
	Parameters []byte `json:"parameters,omitempty"`
	...
}
```